server/.env.example

```diff
@@ -21,7 +21,7 @@ CEREBRAS_API_KEY="your_cerebras_api_key"
 BLOB_STORAGE_BUCKET=ito-blob-storage
 TIMING_BUCKET=ito-timing-storage
 AWS_REGION=us-east-1
-S3_ENDPOINT=http://localhost:9000
+S3_ENDPOINT=http://minio:9000
```

Comment:
I had to make this change to get my stack to work locally -- this is necessary since this is accessed within docker compose